README.md

```diff
@@ -90,6 +90,17 @@ helm install osiris/osiris-edge \
   --devel
 ```
 
+### Configure Osiris
```

Comment:
Something I'm stumbling on if I put myself in the shoes of a new user is that with the previous section titled "Install Osiris" and this section titled "Configure Osiris" these imperatives sound like steps I am supposed to perform _in that order_. Yet, of course, global configuration occurs as _part of_ the installation process.

I think what we can do to improve clarity here would be this:

1. Re-title the previous section as "Installation" (a noun instead of a directive)
2. Re-title this section as " Installation Options"
3. Make installation options a sub-section of "Installation"
4. Make sure the text earlier in the "Installation" section that first introduces the `helm install` command notes that the command installs Osiris with the default configuration and that configuration options are discussed in the next section.

Reply:
good point, I agree that it would be better